CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - ***BREAKING:*** `EsReview` now does not contain padding or margins; responsibility of wrapping element
+- ***BREAKING:*** `bg-white` utility class no longer has an `!important` property
```

Comment:
nit: `bg-white` utility class now includes an `!important` property

Reply:
Actually this isn't to describe the fix in this PR, it's to update the v0.12.0 release changelog (for those that haven't upgraded to it yet) to describe the initial breaking change that I'm fixing in this PR, just so people are aware going in that it may affect things.